Assembly-CSharp/ModHooksGlobalSettings.cs

```diff
@@ -44,6 +44,24 @@ public class ModHooksGlobalSettings
         /// </summary>
         public int PreloadBatchSize = 5;
 
+        /// <summary>
+        ///     When enabled, listens for filesystem changes to mod DLLs.
+        ///     On modification, mods will be unloaded, and the new copy of the assembly gets loaded as well.
+        ///     <para><b>Limitations:</b></para>
+        ///     <list type="bullet">
+        ///         <item><description>
+        ///             The old assembly does not get unloaded. If you have created any unity game objects or components,
```

Comment:
This seems rough, not because of needing to destroy stuff in unload but because it makes a reload costly.